README.md

```diff
@@ -1,21 +1,21 @@
-[![Build status](https://github.com/git/git/workflows/CI/PR/badge.svg)](https://github.com/git/git/actions?query=branch%3Amaster+event%3Apush)
+[![Build status](https://github.com/git/git/workflows/CI/PR/badge.svg)](https://github.com/git/git/actions? query=branch%3Amaster+event%3Apush)
 
 Git - fast, scalable, distributed revision control system
 =========================================================
 
 Git is a fast, scalable, distributed revision control system with an
-unusually rich command set that provides both high-level operations
-and full access to internals.
+unusually rich command set that provides both high-level operations and low-level operations
+and full access to the internals.
```

Comment:
I am not a native speaker, either, but from my understanding, "access to internals" is just as valid as "access to the internals", but it means something subtly different.

As I said, I am not a native speaker, either, therefore I would not try to change it.